pkgs/tools/misc/kalker/default.nix

```diff
@@ -16,14 +17,18 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "sha256-fBWnMlOLgwrOBPS2GIfOUDHQHcMMaU5r9JZVMbA+W58=";
 
-  nativeBuildInputs = [ m4 ];
+  # https://gitlab.com/tspiteri/gmp-mpfr-sys/-/issues/20
+  nativeBuildInputs = [ gcc ];
```

Comment:
gcc is part of the stdenv, isn't it?

Reply:
Not on Darwin. That's the reason for the link above it.

Reply:
I don't want to compile it with gcc on Darwin. I only want to have the gcc binary available, so the build.rs step of `gmp-mpfr-sys` can use it, until the issue has been resolved.